src/tests/calculate.test.js

```diff
@@ -0,0 +1,11 @@
+import calculate from '../logic/calculate';
+
+test('Return null when the button name is "AC"', () => {
+  const input = calculate({}, 'AC');
+  const result = {
+    total: null,
+    next: null,
+    operation: null,
+  };
+  expect(input).toEqual(result);
+});
```

Comment:
- [ ] All the functionalities & scenarios of the `calculate.js` module should be tested, you just test one `AC`, kindly add them 🙃 😉 
   - `buttonName === '0'` should be tested,
   - `buttonName === '.'` should be tested,
   - `buttonName === '='` should be tested,
   - `buttonName === '+/-'` should be tested

Reply:
Thanks a lot @cynthiainga🙏.
But I guessed that it is not clearly mentioned to test all the options for `calculate()` function in the the [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_tests.md). Though I have added those thing you suggested.